index.js

```diff
@@ -0,0 +1,50 @@
+var playItem = require('./src/playitem.js');
+
+var playlistMaker = function(player, plist) {
+  var currentIndex = 0;
+  var playlist = function playlist(list) {
+    this.playlist = playlistMaker(this, list);
+  };
+
+  playlist.list = function list() {
+    return plist.slice();
+  };
+
+  playlist.item = function item(index) {
```

Comment:
We could also add other player level functions instead of trying to overload the playlist property.
```
player.currentPlaylistItem(1); // set
player.currentPlaylistItem(); // get
player.next() or player.nextPlaylistItem();
```
The naming is up for grabs but that feels more consistent with other APIs.